tools/ocp-check-headers/checkHeaders.ml

```diff
@@ -424,6 +464,10 @@ let () =
   let arg_skip_headers = ref StringSet.empty in
 
   let arg_list = Arg.align [
+    "--min-header-lines", Arg.Set_int min_header_lines,
+    " threshold from which a comment block can be a header (3)" ;
```

Comment:
`Arg.align` expects the string to start with the argument, i.e. something like `"NLINES Threshold from which..."`